csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -372,6 +374,8 @@ The resulting translated document:
   It SHOULD NOT use the original `/document/tracking/id` as a suffix.
   If an issuer uses a CSAF translator to publish his advisories in multiple languages they MAY use the combination of
   the original `/document/tracking/id` and translated `/document/lang` as a `/document/tracking/id` for the translated document.
+  > Note that the term "publish" is used in this conformance profile independent of whether the intended target group is the public
```

Comment:
Sorry for the noise, just want to ensure that whatever it will be intended or specified (or even declared) will be written in all scattered places.

Reply:
Nothing to be sorry for - thank you for flagging all places.
[39a992a](https://github.com/oasis-tcs/csaf/pull/873/commits/39a992af59e2a47933a360e9724a3ff682af5e9c)